### DISCUSSION TITLE
Adding a "Noise" effect filter for sound bus

### DISCUSSION THREAD

**Discussion by bugbountyguy** [godotengine/godot-proposals#10459] [Editor]
Hi there! I hope this is the right category for this, I didn't see one for audio/sound.

I've been playing with manipulating some custom sound buses for my game. I have a use case in my game where I _want_ to add static to some specific channels, and this appears to be rather difficult to do with the current effects. Distortion is as close as I can get to it, where I amplify the sound a bunch to get it kind of more static-like but then apply a negative Amplify effect to make sure it doesn't blow any players ears out.

Would it be possible to add an effect to simply add "noise" to get this static-like behavior in? Think static like a radio static or old television static station. 

Thanks!

**Comment by AThousandShips** [comment]
See also:
* https://github.com/godotengine/godot-proposals/issues/9777
* https://github.com/godotengine/godot-proposals/issues/3394

**Reply by bugbountyguy** [reply]
ooooh nice, didn't think of looking for "Procedural" in my searches. Also didn't get the results back for noise audio from github interestingly enough. Thank you for linking those!!
